packages/astro/src/core/build/index.ts

```diff
@@ -210,6 +248,20 @@ class AstroBuilder {
 		});
 		this.logger.debug('build', timerMessage('Additional assets copied', this.timer.assetsStart));
 
+		createInjectedAssets({ settings: this.settings }, this.logger);
+		// console.log('DEBUG', this.settings.resolvedInjectedAssets);
+		this.logger.info('assets', 'Injecting assets...');
+		const assetsPath = this.settings.config.build.assets;
+		for (const resolvedAsset of this.settings.resolvedInjectedAssets) {
+			const assetURL = new URL(
+				`./${assetsPath}/${basename(fileURLToPath(resolvedAsset.resolvedEntryPoint))}`,
```

Comment:
If the `assetsPath` is used, it will likely always be under `_astro`. I think that’s probably correct, but goes against one of your stated goals that the URL could be anything.

Reply:
Also should we rely on the resolved filepath to determine the asset name? I’d expect integration authors to refine the hosted path.